src/components/Discover/TrendingTokens.tsx

```diff
@@ -0,0 +1,622 @@
+                <View
+                  style={{
+                    height: 12 + 2,
+                    width: 12 + 2,
+                    borderRadius: 6,
+                    borderWidth: 1,
+                    borderColor: colors.dark,
+                    backgroundColor: colors.alpha(colors.dark, 0.6),
+                    marginVertical: -1,
+                    marginLeft: -6,
+                  }}
+                />
+                <View
+                  style={{
+                    height: 12 + 2,
+                    width: 12 + 2,
+                    borderRadius: 6,
+                    borderWidth: 1,
+                    borderColor: colors.dark,
+                    backgroundColor: colors.alpha(colors.dark, 0.4),
+                    marginVertical: -1,
+                  }}
+                />
+              </View>
+
+              <FakeText width={148} />
+            </View>
+
+            <View style={{ flexDirection: 'row', gap: 12, alignItems: 'center' }}>
+              <View style={{ gap: 8 }}>
+                <View style={{ flexDirection: 'row', gap: 6, alignItems: 'baseline' }}>
+                  <FakeText height={16} width={84} />
+                  <FakeText height={14} width={32} />
+                  <FakeText height={16} width={60} />
+                </View>
+
+                <View style={{ flexDirection: 'row', gap: 18, alignItems: 'center' }}>
+                  <View style={{ flexDirection: 'row', gap: 4 }}>
+                    <FakeText height={14} width={32} />
+                    <FakeText height={14} width={44} />
+                  </View>
+
+                  <View style={{ flexDirection: 'row', gap: 4 }}>
+                    <FakeText height={14} width={36} />
+                    <FakeText height={14} width={52} />
+                  </View>
+                </View>
+              </View>
+
+              <View style={{ gap: 8, marginLeft: 'auto' }}>
+                <View style={{ flexDirection: 'row', gap: 2, alignItems: 'center' }}>
+                  <FakeText height={16} width={60} />
+                </View>
+                <View style={{ flexDirection: 'row', gap: 5, justifyContent: 'flex-end' }}>
+                  <FakeText width={40} />
+                </View>
+              </View>
+            </View>
+          </View>
+        </View>
+      </Skeleton>
+    </View>
+  );
+}
+
+function TrendingTokenRow({ item }: { item: TrendingTokensType['trendingTokens']['data'][number] }) {
+  const separatorColor = useForegroundColor('separator');
+
+  const marketCap = useMemo(
+    () => formatNumber(item.market.market_cap?.value || 0, { useOrderSuffix: true, decimals: 1 }),
+    [item.market.market_cap?.value]
+  );
+
+  const isPositiveChange = useMemo(
+    () => item.market.price?.change_24h && item.market.price?.change_24h > 0,
+    [item.market.price?.change_24h]
+  );
+
+  const price = useMemo(() => `$${item.market.price?.value ?? '0'}`, [item.market.price?.value]);
```

Comment:
Nit but I'm pretty sure the overhead of useMemo outweighs any gains for simple transformations, unless you need a stable reference for a specific reason.